gas/config/tc-riscv.c

```diff
@@ -2546,6 +2554,8 @@ struct option md_longopts[] =
   {"march-attr", no_argument, NULL, OPTION_ARCH_ATTR},
   {"mno-arch-attr", no_argument, NULL, OPTION_NO_ARCH_ATTR},
 
+  /* These options are used for internal usage or testing.  */
+  {"mno-check-constraints", no_argument, NULL, OPTION_NO_CHECK_CONSTRAINTS},
```

Comment:
It is better to add options in pairs.  Someone will always find a reason to use the other one.  For instance, if you assemble with -mno-check-constraints, and then want to assemble again with constraints checks enabled, then you can just use "!! -mcheck-constraints", but if you don't have the positive option you have to edit the last command which takes more time then just adding another option at the end.

Reply:
Agree with this, thanks :)